### DISCUSSION TITLE
Welcome to video_loopback_for_webui Discussions!

### DISCUSSION THREAD

**Discussion by fishslot** [fishslot/video_loopback_for_webui#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xianienie** [comment]
很好用的插件！♪(^∇^*) 配合controlnet可以有很好的效果，但是使用controlnet时只能使用原视频或图像跑一遍算法来生成控制图像，效果不是很好。是否能够加入专门用于controlnet的视频，比如描边视频，法线图视频 让controlnet直接使用这些视频帧 (*/ω＼*)

**Reply by fishslot** [reply]
这个功能应该已经做好了啊？用不了吗？你试试更新到最新版看看，在reference_frames那里填你的参考帧路径。如果你已经把参考帧提取过描边啥的了，记得在control net的面板里面把预处理设为none。

**Reply by xianienie** [reply]
大佬更新一下，使用多个controlnet分别选择不同的reference_frames

**Reply by fishslot** [reply]
目前controlnet插件的多网络功能还不支持外部控制，等那边更新。那边已经提了issue了，还不知道他会怎么改，应该快了

**Comment by FizzleDorf** [comment]
Will you make this available to use in batch img2img so ControlNet can use the input off of that as well as this script? The alternative would be on ControlNet to make it so you are able to use batch images in it's own input. What do you think?

**Reply by fishslot** [reply]
This is already supported. https://github.com/fishslot/video_loopback_for_webui#news, enable the controlnet, then just use it normally.

**Comment by takahiroanno** [comment]
awesome plugin! I did some experiment using this plugin. just FYI
https://twitter.com/takahiroanno/status/1629122105521627136

**Comment by blade2019runner** [comment]
晕，安装好了，愣是没有找到在哪里用

**Reply by blade2019runner** [reply]
是直接用Batch img2img生成就可以自动识别Video吗？
我这显示cannot identify image file 'C:\\Users\\Administrator\\Desktop\\temp\\MJ\\Video\\Clip.mp4'
ffmpeg也装了

**Reply by fishslot** [reply]
不是，就是直接在img2img界面或者inpainting界面用，在脚本里面找 Video Loopback 。

**Reply by blade2019runner** [reply]
晓得了，谢谢大佬

**Comment by amida168** [comment]
Any tutorial? Thanks.

**Reply by fishslot** [reply]
Here is a simplified and possibly slightly outdated tutorial document that you can read using Google Translate https://github.com/fishslot/video_loopback_for_webui/wiki

**Reply by amida168** [reply]
When I tried to run it on two different computers, I got the same error.

Traceback (most recent call last):
  File "/data/stable-diffusion-webui/modules/call_queue.py", line 56, in f
    res = list(func(*args, **kwargs))
  File "/data/stable-diffusion-webui/modules/call_queue.py", line 37, in f
    res = func(*args, **kwargs)
  File "/data/stable-diffusion-webui/modules/img2img.py", line 170, in img2img
    processed = modules.scripts.scripts_img2img.run(p, *args)
  File "/data/stable-diffusion-webui/modules/scripts.py", line 407, in run
    processed = script.run(p, *script_args)
  File "/data/stable-diffusion-webui/extensions/video_loopback_for_webui/scripts/video_loopback.py", line 513, in run
    reference_img_ques = [
  File "/data/stable-diffusion-webui/extensions/video_loopback_for_webui/scripts/video_loopback.py", line 514, in <listcomp>
    TemporalImageBlender(
  File "/data/stable-diffusion-webui/extensions/video_loopback_for_webui/scripts/video_loopback.py", line 34, in __init__
    self.window = deque(
  File "/data/stable-diffusion-webui/extensions/video_loopback_for_webui/scripts/video_loopback.py", line 35, in <genexpr>
    self.read_image_resize(image_path_list[i])
IndexError: list index out of range

**Reply by fishslot** [reply]
check your reference frames path

**Comment by xianienie** [comment]
大佬能更新一下批量里面按照放大倍率来进行分辨率设置吗，比如一个批次里面有800x1000 1280x720 800x600的三张图 现在是统一设置成了一个分辨率  出来就是三张一样的分辨率的图
比如我设置成2倍分辨率 就可以生成 1600x2000 2560x1440 1600x1200的三张图
能更新一下吗？大佬的插件真的很好用
